api_types.ts

```diff
@@ -449,6 +449,16 @@ export interface CommonPackFormulaDef<T extends ParamDefs> {
    * so an end user must have both sets of permissions.
    */
   readonly extraOAuthScopes?: string[];
+
+  /**
+   * A list of regular expressions that match URLs that the formula is capable of handling.
+   * As described in {@link makeFormula}, this is a discovery mechanism.
+   *
+   * This will replace the `matchers` property in {@link Format} in a future version.
+   */
+  readonly matchers?: string[];
```

Comment:
This should be `RegExp[]` here where the maker specifies it, matching the typing for column format matchers, right? It's only in the compiled metadata that we'd type this as `string[]`, and the compiler serializes it into a string.

@patrick-codaio did a bunch of iterations on this way back and I think after several bugs we landed at the declaration as a RegExp being the way to go.

Reply:
cool thanks i was getting confused about the two different types here